gbp/scripts/pull.py

```diff
@@ -226,7 +220,7 @@ def main(argv):
         gbp.log.err("Interrupted. Aborting.")
     except CommandExecFailed:
         retval = 1
-    except (GbpError, GitRepositoryError) as err:
```

Comment:
You can keep `GbpError` here as other parts of the stack might raise this.

Reply:
done.